packages/block/src/helpers.ts

```diff
@@ -158,7 +164,7 @@ export const getDataGasPrice = (header: BlockHeader) => {
  * @returns the total data gas fee for a transaction assuming it contains `numBlobs`
  */
 export const calcDataFee = (numBlobs: number, parent: BlockHeader) => {
```

Comment:
this fn has been refactored in https://github.com/ethereumjs/ethereumjs-monorepo/pull/2750 as per latest specs and will need to be rebased